explorer/templates.go

```diff
@@ -107,6 +109,50 @@ var toInt64 = func(v interface{}) int64 {
 	}
 }
 
+// standard golang package math in go1.9 doesn't support math.Round
+func Round(val float64, places int) (newVal float64) {
+	pow := math.Pow(10, float64(places))
+	digit := pow * val
+	newVal = math.Floor(digit) / pow
+	return
+}
+
+// boldNumPlaces defines the number of decimal places to be write with same font as the whole
```

Comment:
Please lead the comment with

`// float64Formatting formats a float64 value in multiple strings: the digits on the left of the decimal place, the....`

Reply:
ok. Noted